File: QuanLyPhongKham.Server/src/QuanLyPhongKham.DTOs/Responses/LichLamViecResponse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace QuanLyPhongKham.DTOs.Responses
{
    public class LichLamViecResponse
    {
        public Guid MaLLV { get; set; }
        public DateTime NgayLamViec { get; set; }
        public DateTime GioBatDau { get; set; }
        public DateTime GioKetThuc { get; set; }
        public Guid? MaBS { get; set; }
        public string? TenBacSi { get; set; }
    }
}

File: QuanLyPhongKham.Client/src/interfaces/responses/LichLamViecResponse.js
/**
 * @typedef {Object} LichLamViecResponse
 * @property {string} maLLV
 * @property {string|Date} ngayLamViec
 * @property {string|Date} gioBatDau
 * @property {string|Date} gioKetThuc
 * @property {string} maBS
 * @property {string} tenBacSi
 */

export const createLichLamViecResponse = (
  maLLV = "",
  ngayLamViec = null,
  gioBatDau = null,
  gioKetThuc = null,
  maBS = null,
  tenBacSi = "",
) => ({
  maLLV,
  ngayLamViec,
  gioBatDau,
  gioKetThuc,
  maBS,
  tenBacSi,
});
